src/validate.ts

```diff
@@ -189,6 +194,9 @@ export function response(compiledPath: CompiledPath | undefined,
   if (response === undefined) {
     response = operation.responses['default'];
   }
-
+  // make Date or ObjectId to string
```

Comment:
OK, I'll just remove these  code, but I think return more error detail would still be a good idea, can I keep this?